packages/api/src/routers/api/team.ts

```diff
@@ -158,4 +158,48 @@ router.get('/tags', async (req, res, next) => {
   }
 });
 
+router.delete('/users/:userEmail', async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    const userEmail = req.params.userEmail;
+    if (teamId == null) {
+      throw new Error(`User ${req.user?._id} not associated with a team`);
+    }
+    if (userEmail == null) {
+      throw new Error(`User email not provided`);
+    }
+
+    const team = await getTeam(teamId);
+    if (team == null) {
+      throw new Error(`Team ${teamId} not found`);
+    }
```

Comment:
any reason we need to look up team here ?

Reply:
Thanks for the review, changes made